src/SwordBundle.php

```diff
@@ -15,7 +15,11 @@ public function getPath(): string
         return \dirname(__DIR__);
     }
 
-    public function build(ContainerBuilder $container)
+    /**
+     * @param ContainerBuilder $container
+     * @return void
+     */
```

Comment:
Thanks for your PR! This docblock seems unnecessary, I believe it can be safely removed.